src/index.ts

```diff
@@ -43,9 +43,8 @@ const PluginsHandler = {
 
     if (!plugins[cleanedProperty]) {
       throw new Error(`
-        The ${cleanedProperty} Plugin or Network has not been included in your bundle. Please
-        manually include the script tag associated with this plugin or network. You can see
-        documentation on https://adjs.dev.
+        The Plugin or Network has not been included in your bundle.
```

Comment:
Are we not able to display the missing plugin anymore?

Reply:
I wasn't able to generate it dynamically in the error file - if you have an idea for how to do this/whether we should, I'd be interested!

Reply:
Can't say for certain but it may have to do with the issue i filed on the repo.
I had to switch some of the template literal to the o.g. concat for the moment